template/server/messaging.py

```diff
@@ -27,6 +31,9 @@
 
 logger = logging.getLogger(__name__)
 
+MAX_RECONNECT_RETRIES = 3
+PING_TIMEOUT = None
```

Comment:
why None?

Reply:
Sorry, didn't want to commit this, my bad